python/ray/_private/BUILD

```diff
@@ -2,5 +2,6 @@ load("//bazel:python.bzl", "doctest")
 
 doctest(
     files = glob(["**/*.py"], exclude=["**/thirdparty_files/**"]),
+    size = "large",
```

Comment:
Do we need this?

Reply:
Yes unfortunately. In order to get accurate numbers, I do a large number of iterations, and this takes longer than 300s (the medium test timeout).

Reply:
Ah I think it shouldn't be necessary though, if we only run in the nightly release tests instead of in CI?

Reply:
Yes, I will take this out now.